### PR TITLE
[WEB-970] Ensure export is in scope of variable and window is not required

### PR DESCRIPTION
**Pull request by somewhatabstract**
Here I have done the following:

1. Updated `download.sh` to work when the repo is cloned as a sibling to webapp
2. Executed the new `download.sh` to update the files
3. Updated `add_module_exports.py` to add the `window` existence check and to put the `module.exports` line inside the IIFE so that our build will scope it correctly
4. Executed the new `add_module_exports.py` to update all the files

Address WEB-970

The commits are broken down as follows:

1. Updated the scripts (this is the code changes)
2. Downloaded new files
3. Updated the files by running the updated script

I recommend looking at the initial commit and then comparing one or two files between the 2nd and 3rd commits to see how the script modified them.

### CODE REVIEW

File: add_module_exports.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def main():
     files_not_updated = []
 
     for file in files:
-        if file.endswith(".js"):
+        if file.endswith(".js") and not file.endswith("__language__.js"):
```
Comment (somewhatabstract) [inline add_module_exports.py:28]: Yeah, this was a tweak to avoid an annoying repeated error.

**Review by jeresig — APPROVED** [review]
Yay! Very nice! Thank you for updating the script!

File: icu.__language__.js [hunk 2]
```diff
@@ -1 +1 @@
-icu.en.js
```
Comment (somewhatabstract) [inline icu.__language__.js:1]: Hmm, I don't think this change should occur, but I don't know what caused it. I'll probably revert this file and worry about what changed it another time.

**Comment by somewhatabstract** [comment]
@john can you take a look at the three most recent commits. I noticed some things that needed updating.

1. Some files that apparently weren't redownloaded needed the new export format. I suspect we could delete them instead, but I want to make the webapp update as risk-free as possible, so I made them work the new way instead.
2. The download.sh was symlinking the wrong file path for `icu.__language__.js`. Though I suspect that means this file is not used at all by webapp anymore, I erred on the side of caution and fixed it.
3. The README needed a change to reflect the updated scripts

**Review by jeresig — APPROVED** [review]
Great, thank you for the additional fixes!

**Pull request merged**